src/Network/NWPathMonitor.cs

```diff
@@ -33,6 +41,12 @@ public NWPathMonitor ()
 		public NWPathMonitor (NWInterfaceType interfaceType)
 		{
 			InitializeHandle (nw_path_monitor_create_with_type (interfaceType));
+			InternalSetUpdatedSnapshotHandler ((path) => {
```

Comment:
Could we move the lambda to a method, that way we don't need to write the same code twice.